wiki/advantages-and-features.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Gridcoin Advantages & Features
+layout: wiki
+---
+
+
+# Competitive Advantages 
+
+### General Advantages
+
+One of the main advantages to Gridcoin is that crunching ("mining")
```

Comment:
```suggestion
One of the main advantages of Gridcoin is that crunching ("mining")
```

Reply:
To clarify, both are grammatically correct. However using "of" sounds a bit more natural, so I implemented this

Reply:
Thanks! Didn't know that.